# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/base/PresShell.h
# Commit: 09de143215a1
# Full Hash: 09de143215a13a22dcb2cd53d4ed1dfb60480c82
# Author: Atila Butkovits <abutkovits@mozilla.com>
# Date: 2025-09-30 02:48 +0300
# Regressor Bug: 1987945
# File Overlap Count: 2
# Overlapping Files: accessible/base/nsCoreUtils.cpp, accessible/tests/browser/relations/browser_anchor_positioning.js
# Description:
#   Revert "Bug 1987945: apply code formatting via Lando" for causing bustages at DocAccessible.cpp.
# ==============================================================================

diff -r 826eb4ac00dd -r 09de143215a1 layout/base/PresShell.h
--- a/layout/base/PresShell.h	Tue Sep 30 02:35:22 2025 +0300
+++ b/layout/base/PresShell.h	Tue Sep 30 02:48:50 2025 +0300
@@ -774,18 +774,9 @@
   }
 
   inline void RemoveAnchorPosPositioned(nsIFrame* aFrame) {
-#ifdef ACCESSIBILITY
-    if (nsAccessibilityService* accService = GetAccService()) {
-      accService->NotifyAnchorPositionedRemoved(this, aFrame);
-    }
-#endif
     mAnchorPosPositioned.RemoveElement(aFrame);
   }
 
-  const nsTArray<nsIFrame*>& GetAnchorPosPositioned() const {
-    return mAnchorPosPositioned;
-  }
-
 #ifdef MOZ_REFLOW_PERF
   void DumpReflows();
   void CountReflows(const char* aName, nsIFrame* aFrame);